theseus/utils/sparse_matrix_utils.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+from scipy.sparse import csr_matrix, lil_matrix
+
+
+def generate_mock_sparse_matrix(rows, cols, fill, min_entries_per_col) -> csr_matrix:
```

Comment:
How about `generate_random_binary_matrix`? This module is more general than unit tests, so having "mock" generating routines here seems somewhat out of place.

Reply:
`generate_random_sparse_binary_matrix`?

Reply:
Hehe, yes, I was just realizing that this proposal leaves out the sparse property and came to comment on it. This name is a bit too long, though.

Reply:
that's good for me!

Reply:
Maybe `rand_sparse_binary_matrix`?

Reply:
You could drop random since 'generate' implicitly implies that.